entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/Agent.java

```diff
@@ -67,13 +95,11 @@ public Agent() {
 
 	private List<String> exactMatch;
 
-
 	@JsonGetter(WebEntityFields.WAS_PRESENT_AT)
```

Comment:
the constants are now statically imported the WebEntityFields should be removed in annotations as well

Reply:
Improved.